ui/job-view/headerbars/FiltersMenu.jsx

```diff
@@ -100,26 +98,35 @@ function FiltersMenu(props) {
           <li
             title="Pin all jobs that pass the global filters"
             className="dropdown-item"
-            onClick={pinAllShownJobs}
           >
-            Pin all showing
+            <button
```

Comment:
Using a button element is fine, but the advantage of using the react-strap `Button` component is that you don't need to use the `type=button` attribute. And since you used it in the second file, I'm wondering why you didn't do that here also :)

Reply:
You're right, there is no reason why I didn't change these ones too.